docs/get-started/cardano-node/installing-cardano-node.md

```diff
@@ -334,23 +347,34 @@ sudo ln -s /opt/homebrew/opt/openssl@3/include /usr/local/opt/openssl/include
 
 This is a wart of the `HsOpenSSL` library wrapper, and using classic methods such as setting `LDFLAGS` & `CPPFLAGS`, or using `--extra-include-dirs` and `--extra-lib-dirs` won't work properly.
 
-### Building and installing the node
+:::
+
+##### Building and installing the node
 
 Build the node and CLI with `cabal`:
 
 
 ```bash
 cabal update
-cabal build all
-cabal build cardano-cli
+cabal build exe:cardano-node
```

Comment:
The mention above says cardano-cli too, which cant be built from node repo without referencing external repo (or using cabal install)

Reply:
Good catch :+1: